src/cron/index.ts

```diff
@@ -37,4 +38,5 @@ export const crons: Cron[] = [
   updateSourcePublicThreshold,
   calculateTopReaders,
   validateActiveUsers,
+  cleanGiftedPlus,
```

Comment:
You have to setup cron in `.infra/crons`

Reply:
Thanks! 🙏

Reply:
Updated ✔️